drakrun/drakrun/main.py

```diff
@@ -265,18 +275,22 @@ def process(self):
             self.log.info("note that artifacts will be stored under this overriden identifier")
 
         workdir = '/tmp/drakrun/vm-{}'.format(int(INSTANCE_ID))
+
         extension = self.current_task.headers.get("extension", "exe").lower()
-        start_command = 'start malwar.{}'.format(extension)
+        if '(DLL)' in magic_output:
+            extension = 'dll'
+        self.log.info("Running file as  %s", extension)
 
-        if extension == 'dll' or '(DLL)' in magic_output:
-            run_command = self._get_dll_run_command(sample.content)
+        file_name = self.current_task.payload.get("file_name", f"malwar.{extension}")
```

Comment:
This is user input so maybe we could have some (at least) very simple validation? Sometimes there may be a glitch in some script which may yield some very strange file name with NULLs and special characters (e.g. if the filename is derived from some memory dump, network communication or sth) and this might screw up the sandbox in some strange way or cause command injection